doc/whats_new.rst

```diff
@@ -156,6 +156,15 @@ Enhancements
      A ``TypeError`` will be raised for any other kwargs. :issue:`8028`
      by :user:`Alexander Booth <alexandercbooth>`.
 
+   - Added ``_required_parameters`` class attribute to all estimators that are not
+     default-constructible to specify required parameters programatically.
+     Models with ``_required_parameters`` will not be tested using the
+     common tests, unless the only required parameter is called ``estimator``,
+     in which case the test will be run with a scikit-learn estimator. By `Andreas Müller`_.
+
+   - Added the ``_get_tags`` method to all estimators to return estimator
+     tags that describe estimator capabilities for automated testing. By `Andreas Müller`_.
```

Comment:
suggestion:

[...] to all estimators. The returned :ref:`estimator_tags` describe an object's capabilities and are useful for purposes such as automated testing.

(i like crossrefs, and you are repeating estimator 3 times in a single sentence